conf/nomad/mc.hcl

```diff
@@ -35,7 +35,7 @@ job "${service_name}" {
     %{ endif }
 
       config {
-        image = "${image}"
+        mc_container_image = "${mc_container_image}"
```

Comment:
This won't work. In the config object `image` is a set variable. Must be `image = "${ ... }`.